openlibrary/plugins/openlibrary/lists.py

```diff
@@ -214,18 +214,10 @@ def is_enabled(self):
 
     def GET(self, path):
         # If logged in patron is viewing their lists page, use MyBooksTemplate
-        if path.startswith("/people/"):
-            user = get_current_user()
-            username = path.split('/')[-1]
-
-            if user and user.key.split('/')[-1] == username:
-                return MyBooksTemplate(username, 'lists').render()
-        doc = self.get_doc(path)
-        if not doc:
-            raise web.notfound()
-
-        lists = doc.get_lists()
-        return self.render(doc, lists)
+        username = path.split('/')[-1]
+        mb = MyBooksTemplate(username, 'lists')
+        template = self.render_template(path, show_header=False)
+        return mb.render(template=template, header_title=_("Lists"), page=mb.user)
```

Comment:
This endpoint isn't only on people pages, it's also used to find the lists a work is on (eg https://openlibrary.org/works/OL257943W/A_Game_of_Thrones/lists )

```suggestion
        if path.startswith("/people/"):
            username = path.split('/')[-1]
            mb = MyBooksTemplate(username, 'lists')
            template = self.render(path, show_header=False)
            return mb.render(template=template, header_title=_("Lists"), page=mb.user)
        doc = self.get_doc(path)
        if not doc:
            raise web.notfound()

        lists = doc.get_lists()
        return self.render(doc, lists)
```